src/components/AddComment.jsx

```diff
@@ -0,0 +1,24 @@
+import { postComment } from "../api";
+import { useState } from "react";
+
+export const AddComment = ({ articleId }) => {
+  console.log(articleId, "<<<");
+  const [newComment, setNewComment] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    postComment(articleId, newComment);
```

Comment:
You should give the user some feedback that their POST has been successful here. At the moment nothing happens and they need to re-fresh the page to see the new comment.